tests/components/persistent_notification/test_notify.py

```diff
@@ -0,0 +1,51 @@
+"""The tests for the notify.persistent_notification platform."""
+import asyncio
+import unittest
+
+from homeassistant.components import notify
+import homeassistant.components.persistent_notification as pn
+from homeassistant.setup import setup_component
+
+from tests.common import get_test_home_assistant
+
+
+class TestPersistentNotify(unittest.TestCase):
```

Comment:
All new tests must be standalone pytest test functions.

Reply:
Ok fair enough. I based this on the group notify tests, so I'll refactor it to be a simple test function.